back/appsync/requirements.txt

```diff
@@ -3,4 +3,5 @@ Jinja2==2.10.3
 bleach==3.1.0
 Pillow==6.2.1
 python-slugify==4.0.0
-sentry-sdk==0.14.1
\ No newline at end of file
+sentry-sdk==0.14.1
+raven-python-lambda==0.1.10
```

Comment:
it use old raven client, which is deprecated and last update was in 2018 as i see. Current sentry is sentry-sdk
https://docs.sentry.io/platforms/python/aws_lambda/ here current way. maybe there also package that works

Reply:
Ok, updated integration with newer method